cartridge

```diff
@@ -2038,6 +2098,11 @@ end
 -- is successfully bootstraped: after tarantool executes main script
 -- it sends `READY=1` message to the NOTIFY_SOCKET.
 function cmd_start.callback(args)
+    if args.instance_name == nil then
```

Comment:
Is it possible to run single application without specifying instance name and configuration path using simple command:

```bash
cartridge start myapp
```

Reply:
Yep, there are examples in readme

Reply:
I tried and got an error:

```bash
$ cartridge create --template cartridge --name myapp
$ cd myapp
$ rm myapp-scm-1.rockspec
$ cartridge start myapp
error: Failed to open file /Users/i.konyuhov/Workspace/cartridge-cli/myapp/instances.yml: No such file or directory
```

Reply:
You need to create configuration file with instances list. Otherwise it can not know which instances do you want it to start.

Reply:
I just want to start single application (maybe test something). Should it be necessary to write additional configuration file (or provide additional cmd arguments) for such case?

Reply:
`$ ./init.lua` should do the work

Reply:
`cartridge start .instance_name` should also work